lib/theming/src/emotionAugmentation.d.ts

```diff
@@ -0,0 +1,9 @@
+// this file is only actually used in development
```

Comment:
Believe it or not but this has to be in a separate file and I can't reuse `src/typings.d.ts` - I've already stumbled upon a problem like this in the past, TS rules are weird when it comes to those `d.ts` files and I don't fully understand them. When put in the other file I couldn't use `import` at the top-level and without it, the tsup didn't recognize other Emotion exports (such as `keyframes`, `css` etc). This way I only touch what I actually want and leave the rest as-is